docs/features/software-templates/adding-templates.md

```diff
@@ -15,7 +18,8 @@ metadata:
   # title of the template
   title: React SSR Template
   # a description of the template
-  description: Next.js application skeleton for creating isomorphic web applications.
+  description:
+    Next.js application skeleton for creating isomorphic web applications.
```

Comment:
Does this newline work? I think there needs to be a `|` or `>` after the colon on the previous line, no?

Reply:
I think this is prettier interference :) But good point, need to double check